### DISCUSSION TITLE
How are the model files (ggml-*****.en.bin) maintained and controlled?

### DISCUSSION THREAD

**Discussion by RndyP** [ggml-org/whisper.cpp#401] [General]
Are the model files under some sort of release procedure? Are they updated? If so, what is the update procedure? Are they set in concrete forever?

I'm asking because I would think any updates would cause different results in the inference, and this will change any test scripts and benchmark results. Suppose we change any code that effect results. If the model files are the same, I assume we can verify the latest and greatest ggml.c by running a set of .wav files and doing a text comparison of the output against previous results, and looking for an exact match (if the software change was not meant to be an improvement). If the model files change, any test scripts may need to be redone.

Another reason I'm asking, is because I am using Whisper primarily as a command interpreter, and I am building a correction dictionary of words that Whisper gets wrong, and this effort may be changed in the future if the model files are updated.

**Comment by ggerganov** [comment]
The model files will change only if OpenAI release new models.
Also, if we find bugs in the conversion process (like #25).
